setup.cfg

```diff
@@ -23,6 +23,7 @@ packages = find:
 install_requires =
     auditree-framework>=1.2.3
     auditree-harvest>=1.0.0
+    PyYAML>=5.3.1,<5.4
```

Comment:
Why?

Reply:
I removed the version constraint.

Originally, this fetcher is stored in auditree-central, and installation (`make install`) failed without this version constraint.  Now we are in arboretum, and this constraint no longer requires. thanks for pointing out this.